### DISCUSSION TITLE
Does Grule supports Timer/ Calendar rule attributes?

### DISCUSSION THREAD

**Discussion by denver1111** [hyperjumptech/grule-rule-engine#313] [Q&A] [answered]
Grule is inspired by Drools, but I don's see any supports for "Duration", "Calendar", "Timer" attributes in Grule, all this are supported in Drools.
Do Grule supports Timer/ Calendar rule attributes as mentioned above?
Or is there any plan to support this attributes in future?

Currently I see only "salience" attribute is supported by Grule. Is there any other attributes present which is supported by Grule as of today?

**Comment by newm4n** [comment, marked as answer]
Hi @denver1111 ,

"Inspired" does not means to "Imitate", Drools indeed inspires Grule but it does not means to copy Drools in its entirety. Its not and will not do that. We are open for contribution though. Would be glad if you could contribute those features that you feel should make Grule better.
